machine_learning/dbscan/dbscan.ipynb

```diff
@@ -0,0 +1,318 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(X[:,0], X[:,1],'ro')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The implementation is inspired from the original DBScan algorithm as given in \n",
+    "<a href = \"https://en.wikipedia.org/wiki/DBSCAN\">DBSCAN Wikipedia</a>\n",
```

Comment:
Can we make that link work when the file is viewed on GitHub?

Reply:
Done.